contracts/evoting/evoting.go

```diff
@@ -700,6 +700,9 @@ func (e evotingCommand) combineShares(snap store.Snapshot, step execution.Step)
 
 	conn.SetWriteDeadline(time.Now().Add(writeTimeout))
 
+	bytesToSend := hex.EncodeToString(buf.Bytes())
+	dela.Logger.Info().Msgf("Data to Unikernel: %v", bytesToSend)
```

Comment:
You might want to use the `.Hex()` function, something like dela.Logger.Info().Hex("out", buf.Bytes()).Msg(...).

Reply:
Same bellow.

Reply:
That works